src/build-clean.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+# @file build-clean.sh
+# @author Jan Rodak - xrodak00
+# @brief Skript na vycisteni vygenerovaných souboru build skriptem  
```

Comment:
U doxygen komentářů dodržujeme konvenci 
""" 
@file bla bla
@author bla bla
atd...
"""
Přepiš to pls